.github/workflows/canopeum_deploy.yml

```diff
@@ -0,0 +1,105 @@
+name: canopeum_frontend_deploy
```

Comment:
Just to give a better name since its merging frontend and backend

Reply:
Good catch! Accidental leftover.